cellmlmanip/model.py

```diff
@@ -599,18 +599,19 @@ def _set_variable_type(variable, variable_type):
             logger.warning('Variable %s already has type=="%s". Skip setting "%s"',
                            variable.dummy, variable.type, variable_type)
 
-    def get_symbols(self, expr):
-        """Returns the symbols in an expression. Works differently to Sympy atoms() because a
+    def get_symbols(self, expressions):
```

Comment:
Should probably be called something like `get_symbols_for` or `get_symbols_for_expression`. I'd expect `model.get_symbols()` to return the symbols in the model

Reply:
Makes sense, done